### DISCUSSION TITLE
Looking for Budibase Freelancer

### DISCUSSION THREAD

**Discussion by YamatoWerner** [Budibase/budibase#12120] [Help]
Hey guys, 

I'm working on a new project and am looking for a budibase freelancer. I checked on Fiverr but couldn't find anyone so I thought I'd just ask if I could find someone here.

It's a pretty simple project,

- Home page with daily question
- Database to store the answers without me being able to see the answers
- A way to access the answers without seeing them in order to send them back to the user

If you are interested, let me know and we can discuss the details, time frame and pay, etc...

**Comment by A3DataSolutions** [comment]
Hi @YamatoWerner,

Feel free to reach out. My website is A3datasolutions.com.

Best,

J
